pages/api/user/profile.ts

```diff
@@ -11,15 +11,11 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   }
 
   // Get user
-  const user = await prisma.user.findUnique({
-    where: {
-      email: session.user.email,
+  const user = await whereAndSelect(prisma.user.findUnique, {
+      id: session.user.id,
     },
-    select: {
-      id: true,
-      password: true
-    }
-  });
+    [ "id", "password" ]
```

Comment:
Codacy has a fix for the issue: [Replace `·"id",·"password"·` with `"id",·"password"`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734611)
```suggestion
    ["id", "password"]
```